systemd/tests/conftest.py

```diff
@@ -0,0 +1,30 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import pytest
+
```

Comment:
can we have `e2e` / `dd-environment` on linux machine at least where we just spin up the docker agent listening to the host systemd?